From d33b9e82f7db632d0d025fa63352ce885061e363 Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Wed, 18 Nov 2015 19:59:35 +0100
Subject: [PATCH] set device specific version

---
 version.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/version.h b/version.h
index 5b34d63..6de2e74 100644
--- a/version.h
+++ b/version.h
@@ -17,9 +17,9 @@
 
 #ifndef VERSION_H
 #define VERSION_H
-    #define VERSION_MULTIROM 32
+    #define VERSION_MULTIROM 32
     #define VERSION_TRAMPOLINE 26
 
     // For device-specific fixes. Use letters, the version will then be like "12a"
-    #define VERSION_DEV_FIX ""
+    #define VERSION_DEV_FIX "m"
 #endif
-- 
2.5.0

