From 9c606f9d482109fbb85140b0bfde71ae4b32c7ff Mon Sep 17 00:00:00 2001
From: nkk71 <nkk71x@gmail.com>
Date: Thu, 8 Oct 2015 16:00:37 +0300
Subject: [PATCH 1/2] RE-Enable MTP during startup

---
 twrp.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/twrp.cpp b/twrp.cpp
index 14d0ad8..e476a45 100644
--- a/twrp.cpp
+++ b/twrp.cpp
@@ -339,7 +339,7 @@ int main(int argc, char **argv) {
 			MultiROM::executeCacheScripts();
 	}
 
-#if 0
+#if 1 // "#if 0" why was this disabled?
 #ifdef TW_HAS_MTP
 	// Enable MTP?
 	char mtp_crash_check[PROPERTY_VALUE_MAX];
-- 
2.5.0

